src/ocm/components/clusterConfiguration/operators/LvmCheckbox.tsx

```diff
@@ -8,29 +11,36 @@ import {
   operatorLabels,
   OperatorsValues,
   FeatureSupportLevelBadge,
+  LVMS_LINK,
+  OPERATOR_NAME_LVMS,
+  ExposedOperatorName,
 } from '../../../../common';
 import LvmHostRequirements from './LvmHostRequirements';
 import { OcmCheckboxField } from '../../ui/OcmFormFields';
 import { useTranslation } from '../../../../common/hooks/use-translation-wrapper';
 import { getActualLVMOperatorName } from './utils';
-import { useFormikContext } from 'formik';
 import { getLvmIncompatibleWithCnvReason } from '../../featureSupportLevels/featureStateUtils';
 
 const LVM_FIELD_NAME = 'useOdfLogicalVolumeManager';
 
-type LvmLabelProps = ClusterOperatorProps; // & { operator: ExposedOperatorName };
-
-const LvmLabel = ({ openshiftVersion, clusterId }: LvmLabelProps) => {
-  const { t } = useTranslation();
-  const [operator, setOperator] = React.useState('');
-
-  const featureSupportLevel = useFeatureSupportLevel();
-  React.useEffect(() => {
-    setOperator(getActualLVMOperatorName({ openshiftVersion, featureSupportLevel }));
-  }, [featureSupportLevel, openshiftVersion]);
+type LvmLabelProps = ClusterOperatorProps & { operatorName: ExposedOperatorName };
 
-  const operatorLabel = operatorLabels(t)[operator];
+const LvmHelperText = ({ operatorName }: { operatorName: ExposedOperatorName }) => {
+  return (
+    <>
+      Storage virtualization that offers a more flexible approach for disk space management.{' '}
+      {operatorName === OPERATOR_NAME_LVMS && (
```

Comment:
@jgyselov There will be conflicts with your PR https://github.com/openshift-assisted/assisted-ui-lib/pull/1925.

But I still see the issue in `ProgressBarAlerts` has not been addressed, so we cannot merge it yet.

Reply:
I will fix conflicts with that PR and the one about the tooltip after they have been merged.